src/class/api.ga4gh.php

```diff
@@ -211,6 +214,11 @@ function __construct (&$oAPI)
         // To not duplicate code.
         $this->aValueMappings['effect'] = $_SETT['var_effect_api'];
 
+        // Retrieve default genome build.
+        global $_DB;
```

Comment:
Globals can be defined at the top of the function/method.

Reply:
Moved :)